Applications/Spire/Source/Ui/TableView.cpp

```diff
@@ -128,6 +128,7 @@ TableView::TableView(
   m_header_scroll_box->set(ScrollBox::DisplayPolicy::NEVER);
   m_header_scroll_box->setSizePolicy(
     QSizePolicy::Expanding, QSizePolicy::Fixed);
+  m_header_scroll_box->setFocusPolicy(Qt::NoFocus);
```

Comment:
Can you describe why the scroll box's focus policy is set to no focus here? What makes TableView special that the focus policy has to be set here?

Reply:
I don’t think ScrollBoxes within TableView need a strong focus policy. When navigating into or out of TableView using Tab or Shift+Tab, ScrollBox which has a strong focus policy (by default) can receive focus, but since it lacks a visual indicator for its focus state, this can be confusing for users. Moreover, TableView doesn’t require ScrollBox to achieve navigation function via the keyboard. Therefore, I set all ScrollBoxes to have no focus policy to avoid them getting focus.